client/components/theme-preview-modal/style.scss

```diff
@@ -0,0 +1,192 @@
+@import "@wordpress/base-styles/breakpoints";
+@import "@wordpress/base-styles/mixins";
+@import "@wordpress/base-styles/variables";
+
+$break-design-preview: 1024px;
+
+@mixin break-design-preview() {
+	@media (min-width: #{ ($break-design-preview) }) {
+		@content;
+	}
+}
+
+.theme-preview-modal {
+	background-color: #fdfdfd;
+	bottom: 0;
+	left: 0;
+	margin: 0 auto;
+	overflow: hidden;
+	position: fixed;
+	right: 0;
+	top: 0;
+	z-index: z-index("root", ".theme-preview-modal");
+	-webkit-font-smoothing: antialiased;
+
+	@include break-design-preview {
+		padding-inline-start: 32px;
+		padding-inline-end: 32px;
+	}
+}
+
+.theme-preview-modal__navigation {
+	align-items: center;
+	bottom: 0;
+	display: flex;
+	inset-inline-end: 0;
+	inset-inline-start: 0;
+	height: 60px;
+	gap: 24px;
+	margin: 0;
+	padding: 0 20px;
+	pointer-events: none;
```

Comment:
Unfortunately, we need `pointer-events: none` so that the device switcher can be clicked:
![Screen Shot 2023-01-04 at 9 47 17 AM](https://user-images.githubusercontent.com/797888/210469710-85dce8bb-ce75-4efa-9a1d-9cdc6acc7db2.png)

However, we can change `> * { ... }` to ` > a, button { ... }` to be less aggressive. What do you think?

Reply:
Okay, I didn't notice that. Let's leave it as it is 🙂